website/www/site/layouts/index.html

```diff
@@ -154,6 +135,35 @@ <h4 class="calendar-card-small-title">
   </div>
 {{ end }}
 
+{{ define "quotes-section" }}
+  <div class="quotes">
+    <div class="quotes-title">
+      They tried it out
+    </div>
+    <div class="quotes-content">
+      {{ $data := index $.Site.Data .Site.Language.Lang }}
+      {{ range $quote := $data.quotes }}
+          {{ partial "quotes/quote.html" (dict "icon" $quote.icon "text" $quote.text  "logoUrl" $quote.logoUrl) }}
+      {{ end }}
+    </div>
+  </div>
+{{ end }}
+
+{{ define "quotes-mobile-section" }}
+  <div class="quotes-mobile">
+    <div class="quotes-mobile-title">
```

Comment:
Is it necessary to have separate quotes-mobile and quotes-mobile-title classes for this part? Couldn't this be handled by CSS?

Reply:
Thanks for suggestion!